test-base.cfg

```diff
@@ -215,7 +221,7 @@ initialization =
         shell=True, stdout=subprocess.PIPE).stdout.read().strip()
     if '--absolute' in sys.argv: pkgdir = os.path.abspath(pkgdir) ; sys.argv.remove('--absolute')
     if len(sys.argv) < 2: sys.argv.extend([pkgdir])
-    sys.argv.extend(${pycodestyle:args})
+    sys.argv.extend(${pycodestyle-ci-run:args})
```

Comment:
The whole point is that `bin/pycodestyle` should offer the normal entry point of the `pycodestyle` egg so editors and other tools can interface with in the ways they expect ('run the tool on this buffer from some tmp while the user is still typing').

Once that's the normal entry point, the CI-run one should just be a simple shell script with a few extra parametres.